agent/test/unit/com/thoughtworks/go/agent/AgentProcessParentImplTest.java

```
@@ -278,14 +275,10 @@ private File randomFile(final String pathname) {
     }
 
     private void createRandomFile(File agentJar, final String data) {
-        FileOutputStream output = null;
-        try {
-            output = new FileOutputStream(agentJar);
+        try (FileOutputStream output = new FileOutputStream(agentJar)) {
             IOUtils.write(data, output);
         } catch (IOException e) {
-            throw new RuntimeException(e);
-        } finally {
-            IOUtils.closeQuietly(output);
+            e.printStackTrace();
```

Sure. That's fine too. This code swallows the exception, though.